### ISSUE TITLE
Should include library(shiny) at beginning of sample code

### ISSUE THREAD

**Issue by ThoDuyNguyen** [daattali/advanced-shiny#2]
Hi @daattali 

Thank you for the very nice topic. It help me a lot to improve my projects. 

On the other hand, it seems to me that the code should include 

```
library(shiny)
```

If I copy and paste it to RStudio and run without the library, there was error message

```
Error: could not find function "fluidPage"
```

and 

```
Error: could not find function "shinyApp"
```

**Comment by daattali** [comment]
Ah you're right, I always ran the code using the button in RStudio so it automatically loads shiny. I'll add it later, thanks

**Comment by daattali** [comment]
@ThoDuyNguyen I'd gladly accept a PR for this if you have 5 minutes to go through and add that line into every app :)

**Comment by ThoDuyNguyen** [comment]
@daattali : You mean that I will pull the project and add that line for each **app.R**  and then commit it again?

**Comment by daattali** [comment]
you need to send a pull request, you don't have write permissions on my repository. You need to fork this into your own repo, do all the changes, commit, and then request to merge into mine

**Comment by aleruete** [comment]
sorry if I messed something up. I'm pretty bad at handling GitHub
repositories.






Alejandro Ruete
about.me/alejandro.ruete
  <http://about.me/alejandro.ruete>

2016-08-30 11:00 GMT+02:00 Dean Attali <notifications@github.com>:

**Comment by daattali** [comment]
@aleruete you didn't do anything, I don't see any commits by you so don't worry :)

**Comment by ThoDuyNguyen** [comment]
@daattali : please check it. 
Kind regards

**Comment by daattali** [comment]
fixed with PR #3
